backend/src/index.js

```diff
@@ -36,13 +43,13 @@ io.on( 'connection', socket => {
 			socket.session.issues.push( issueKey );
 		}
 
-		const issueUsers = db.get( issueKey );
+		const issueUsers = JSON.parse( await getAsync( issueKey ) );
 
-		if ( !issueUsers.has( message.user.login ) ) {
-			issueUsers.set( message.user.login, { sockets: [], tabs: new Map(), joinedAt: timestamp } );
+		if ( !issueUsers.hasOwnProperty( message.user.login ) ) {
+			issueUsers[ message.user.login ] = { sockets: [], tabs: new Map(), joinedAt: timestamp };
```

Comment:
Why `tabs: new Map()`? You use it as an object later.

Reply:
Already fixed.